.bit/responses/4.1-Week 4 Step 1.md

```diff
@@ -40,73 +40,103 @@ Use **[LiveServer](https://marketplace.visualstudio.com/items?itemName=ritwickde
 ```
 
 </details>
 
-<details>
-<summary>:exclamation: How can I connect my JS file to my HTML?</summary>
-  </br>
-
-Great question! All we have to do is reference it just like we did with the jQuery.
-```html
-<head>
-...
-   <script src="index.js" type="text/javascript"></script>
-</head>
-```
-> Place this directly under your jQuery reference.
-</details>
-
 ## Writing the Event Handler Function!
 On websites, there are many different types of events that can occur. For example:
 - The user selects a certain element or hovers the cursor over a certain element.
 - The user chooses a key on the keyboard.
 - A form is submitted.
 - Curious for more? Look [here](https://www.w3schools.com/js/js_events.asp)
 
-We need to create an event handler function called `getImage()`, which triggers when someone **submits an image**.
+We need to create a function which triggers when someone **submits an image**,
+but first we will create a function which changes content on the page when we 
+submit text.
 
-We are going to do this by using the jQuery `text()` method. The syntax to set the text for a [div](https://www.w3schools.com/tags/tag_div.ASP) is `$(selector).text(content)`. 
+<details>
+<summary>I'm confused on how to do this</summary>
 
-To learn more, read the docs [here](https://www.w3schools.com/jquery/html_text.asp)
+- In your html file, give your form an id
 
+- Change the last input from type ="button" to type = "submit"
 
-<details>
-<summary>I'm confused on how to do this</summary>
+```js
+ <input value="Submit" type="submit" />
+```
 
-- The selector should be `#output`, or the name of the div with a # in front.
+- In index.js, create a variable using the id you just created:
+   
+```js
+const bunnForm = document.getElementById('bunnForm');
+```
+- In your index.js file, create an event listener with an anonymous function.
+Inside of the function, create a variable for the value that was input in 
+the text box.
+   
+ ```js
+bunnForm.addEventListener('submit', function (event) {
+   const username = document.getElementById("username").value
+});
+```
+   
+> :bulb: We are retrieving the value of the "username" text box with this code!
+> :bulb: The document.getElementById() method gets the value of the input text from the HTML.
 
-- The content should be `document.getElementById("username").value + "❤️"`
+- Create a variable to target html div with the id of "output" and use the 
+textContent property to add a "❤" to the value that was input for username.
 
-> :bulb: We are retrieving the value of the "name" text box with this code!
+- :bulb" By default, the website will reload whenever a user presses submit. This leads to our text on the page that we set being cleared away. We can prevent this by adding this to the top of our function.
 
-`$('#output').text(document.getElementById("username").value + "❤️")`
+```js
+event.preventDefault()
+```
+   
+```js
+bunnForm.addEventListener('click', function (event) {
```

Comment:
should this be `submit` instead of  `click`, because right now you are triggering the callback when the form is clicked, not submitted

Reply:
I think prior to re-factoring this, onclick was listening for a button click and then it triggered the function but we don't have an onclick event anymore. ￼Now we are listening for the form to submit. I could redo the event listener to listen for the button click but i thought ￼that would be unnecessary steps to completely change the event listener for ￼the next step.

Reply:
got it! Thanks.